detector/aws_db_instance_default_parameter_group.go

```diff
@@ -7,7 +7,15 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsDbInstanceDefaultParameterGroup(issues *[]*issue.Issue) {
+type AwsDBInstanceDefaultParameterGroupDetector struct {
```

Comment:
__[golint]__
* [golint] exported type AwsDBInstanceDefaultParameterGroupDetector should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->